app/components/preprint-status-banner/template.hbs

```diff
@@ -0,0 +1,42 @@
+<div id="preprint-status" class="container p-md">
+    <div class="flex-container">
+        {{#if model.provider.isPending}}
+            Loading...
+        {{else}}
+            <div class="status-explanation">
+                <i class="fa {{icon}} status-icon" aria-hidden="true"></i>
+                <strong>{{t status}}:</strong>
+                <span>
+                    {{t baseMessage name=(if theme.isProvider theme.provider.name (t "global.brand_name")) reviewsWorkflow=(t workflow)}} {{t bannerContent}}.
```

Comment:
No, this just feels like too much logic in the template. I think it'd be easier to understand at a glance if it were a few lines of js in a computed property that do exactly the same thing (using the i18n service), and the template just had `{{translatedBannerMessage}}` or whatever you want to call it.